tests/apps/actorfeatures/app.go

```diff
@@ -362,6 +364,16 @@ func testCallMetadataHandler(w http.ResponseWriter, r *http.Request) {
 	w.Write(body)
 }
 
+func testEnvHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
This can be handy to validate that the app is running with the expected configuration.